test/integration-test.py

```diff
@@ -532,7 +532,7 @@ def main():
         run_loadtest=False, test_case_filter="")
     args = parser.parse_args()
 
-    if not (args.run_all or args.run_certbot or args.run_chisel or args.custom is not None):
+    if not (args.run_all or args.run_certbot or args.run_chisel or args.run_loadtest or args.custom is not None):
         raise Exception("must run at least one of the letsencrypt or chisel tests with --all, --certbot, --chisel, or --custom")
```

Comment:
Can you update this exception with `--load`.

Reply:
Good catch, fixed in bffb8f2